googleapiclient/errors.py

```diff
@@ -126,7 +126,10 @@ def __init__(self, reason, resp=None, content=None):
     self.reason = reason
 
   def __repr__(self):
+    if self.resp is not None and hasattr(self.resp, 'status'):
```

Comment:
If self.resp is None, getattr(None, 'status') will raise Error. I have changed the code in new commit.